lib/omise/OmiseCapabilities.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+class OmiseCapabilities extends OmiseApiResource
```

Comment:
@jonrandy For the naming, `OmiseCapability` would be more consistency with the rest classes' name.

Reply:
I thought about this when I made it. The object returned details the `capabilities` of the system, not a single capability. I think this makes more sense in the current naming scheme. Calling it by the singular would be confusing and inconsistent IMO

Reply:
there are many different capabilities listed..

Reply:
Precisely

Reply:
I discussed with Shunichi back in mid-December, he agrees with the naming and is going to rename the object in the API, as well as the endpoint